ruby_rails/README.md

```diff
@@ -0,0 +1,194 @@
+# Omise Rails
+
+This is a example project about using Opn Payment.
+
+It has some examples.
+- Credit Card Payment
+- PayPay
+- Google Payment
+
+## How to implement Credit Card Payment
+
+In order to run a charge, you need to implement
+- Create a token for credit card
+- Call Charge API using this token.
+
+Please refer this page.
+
+https://docs.opn.ooo/tokens-api/japan
+
+https://docs.opn.ooo/charges-api/japan
+
+https://docs.opn.ooo/ja/omise-js/japan
+
+
+## How to implement PayPay
+
+There are two ways.
+- Create Source, then create charge using this source.
+- Create Source and Charge at the same time.
+
+Please refer this page.
+
+https://docs.opn.ooo/paypay/japan
+
+
+## How to implement Google Payment
+
+- Add Google payment button.
+- Get Google Payment token.
+- Call Omise Token by using Google Payment Token
+- Call Omise Charge API.
+
+Please refer this page.
+
+https://developers.google.com/pay/api/web/guides/resources/demos?hl=ja
+
+https://docs.opn.ooo/ja/googlepay/japan#part-49380a7c4a55a530
+
+
+# Requirements
+To follow along, you will need:
+
+An Omise account (and your $OMISE_PUBLIC_KEY and $OMISE_SECRET_KEY)
+
+This example project use this Ruby and Rails version
+
+- Ruby 3.0.0
```

Comment:
3.0 end of life is in four months, so maybe we should start with a higher version?

Reply:
@danfowler 
Oh, I think you are right.
I will apply higher ruby.

Reply:
https://github.com/omise/examples/pull/30/commits/1e254aa2596ae8dafdc824a080055002bf2b4214

Reply:
https://github.com/omise/examples/pull/30/commits/26a9b60540d502ff93617c04bc8355152c7567db